Python/read_feh.py

```diff
@@ -135,23 +135,40 @@ def read_header_file(filename:str):
         # Get the first 10 bytes
         first_10_bytes = file.read(10)
 
-        # Check if the first 4 bytes of the first 10 represent the year 2006
-        if int.from_bytes(first_10_bytes[:4], byteorder="little", signed=True) == 2006:
+        ### CONDITIONS TO IDENTIFY INPUT FILE
+        # Check if the 5th and 6th bytes are '\x0D\x0A' representing a new line in the file
+        in_year_byte_suffix = first_10_bytes[4:6]
+        # Check if the first 4 bytes of the first 10 represent a year between 2000 and 2100
+        in_year_bytes = int.from_bytes(first_10_bytes[:4], byteorder="little", signed=True)
+        input_conditions = 2000 <= in_year_bytes <= 2100 and in_year_byte_suffix == b'\x0D\x0A'
+
+        ### CONDITIONS TO IDENTIFY OUTPUT FILE
+        # Check if first 6 bytes represent empty space designated by 0x20202020
+        # Check if the the last 4 bytes are an ASCI code that could represent a year
+        empty_space = first_10_bytes[:6] == b'\x20\x20\x20\x20\x20\x20'
+        # Check if the last 4 bytes could represent an ASCII year
+        ascii_year = first_10_bytes[6:10].decode('ascii')
+        output_conditions = empty_space and ascii_year.isdigit() and 2060 <= int(ascii_year) <= 2200
```

Comment:
The condition for the year is too restrictive; we sometimes run shorter simulations. Also, it's probably unnecessary. I think that the empty space condition is sufficient.

Reply:
Addressed! Thanks for your review :)